File: src/main/java/frc/robot/subsystems/climber/ClimberIORealLeft.java
package frc.robot.subsystems.climber;

import com.revrobotics.CANSparkLowLevel;
import com.revrobotics.CANSparkMax;
import frc.robot.Constants;

public class ClimberIORealLeft extends ClimberIOReal {
    @Override
    protected CANSparkMax getMotor() {
        return new CANSparkMax(Constants.Climber.leftMotorId, CANSparkLowLevel.MotorType.kBrushless);
    }
}

File: src/main/java/frc/robot/subsystems/climber/ClimberIORealLeft.kt
package frc.robot.subsystems.climber

import com.revrobotics.CANSparkLowLevel
import com.revrobotics.CANSparkMax
import frc.robot.Constants

class ClimberIORealLeft : ClimberIOReal() {
    override fun getMotor(): CANSparkMax {
        return CANSparkMax(Constants.Climber.leftMotorId, CANSparkLowLevel.MotorType.kBrushless)
    }
}
